smarts/env/hiway_env.py

```diff
@@ -173,7 +210,7 @@ def scenario_log(self):
             "mission_hash": str(hash(frozenset(scenario.missions.items()))),
         }
 
-    def seed(self, seed: int) -> int:
+    def seed(self, seed=None) -> int:
```

Comment:
What is the reason for this change?

Reply:
It didn't match the `OpenAI` gym's `seed` method header and was giving me a warning in my IDE. But i will revert it back for now.